src/Resources/views/Core/dashboard.html.twig

```diff
@@ -16,35 +16,35 @@ file that was distributed with this source code.
 {% block content %}
 
     {% set has_left = false %}
-    {% for block in blocks.left %}
+    {% for block in blocks.left if not has_left %}
```

Comment:
won't this always be false because of set?

Reply:
`has_left` holds `false` as value until the condition in the next if (`{% if block.roles|length == 0 or is_granted_affirmative(block.roles) %}`) evaluates to `true`, then, the iteration is not required anymore since `has_left` holds `true` in that situation.

Reply:
I didn't see it was a `for` loop, ignore me.